Kitodo/src/test/java/org/kitodo/production/services/catalogimport/MassImportTest.java

```diff
@@ -93,11 +99,34 @@ public void shouldUpdateSeparator() {
     public void shouldPrepareMetadata() throws ImportException {
         MassImportService service = ServiceManager.getMassImportService();
         List<CsvRecord> csvRecords = service.parseLines(CSV_LINES, StringConstants.COMMA_DELIMITER);
-        Map<String, Map<String, String>> metadata = service.prepareMetadata(METADATA_KEYS, csvRecords);
+        Map<String, Map<String, List<String>>> metadata = service.prepareMetadata(METADATA_KEYS, csvRecords);
         Assert.assertEquals("Wrong number of metadata sets prepared", 3, metadata.size());
-        Map<String, String> metadataSet = metadata.get("123");
+        Map<String, List<String>> metadataSet = metadata.get("123");
         Assert.assertNotNull("Metadata for record with ID 123 is null", metadataSet);
-        Assert.assertTrue("Metadata for record with ID 123 does not contain title metadata", metadataSet.containsKey(TITLE));
-        Assert.assertEquals("Metadata for record with ID 123 contains wrong title", "Band 1", metadataSet.get(TITLE));
+        Assert.assertEquals("Metadata for record with ID 123 contains wrong title", 2,
```

Comment:
I think the message does not fit here: when the size of the metadata set is wrong shouldn't that be reflected in the message instead of something about a wrong title?